packages/starlight/translations/pl.json

```diff
@@ -0,0 +1,30 @@
+{
+  "skipLink.label": "Przejdź do głównej zawartości",
+  "search.label": "Szukaj",
+  "search.shortcutLabel": "(Naciśnij /, aby wyszukać)",
+  "search.cancelLabel": "Anuluj",
+  "search.devWarning": "Wyszukiwanie jest dostępne tylko w buildach produkcyjnych. Spróbuj zbudować i uruchomić aplikację, aby przetestować lokalnie.",
+  "themeSelect.accessibleLabel": "Wybierz motyw",
+  "themeSelect.dark": "Ciemny",
+  "themeSelect.light": "Jasny",
+  "themeSelect.auto": "Auto",
```

Comment:
Just to double-check, this is expected to stay the same as the English version and something missed?

Reply:
Correct, _Automatic_ – _Automatyczny_.